src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/test/TestBase.java

```diff
@@ -195,9 +216,70 @@ protected void quit() {
 		}
 	}
 	
+	protected WebElement getElement(By elementBy) {
+		try {
+			return driver.findElement(elementBy);
+		} catch (NoSuchElementException e) {
+			return null;
+		}
+	}
```

Comment:
This util already exists in the parent Page class.
we are even not supposed to use this anywhere in the step definitions.
This is supposed to be wrapped up in the Page objetcs

Reply:
Resolved thanks